Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3
+
+ENV KF_INGEST_APP_MODE production
+
+WORKDIR /app
+
+COPY setup.py requirements.txt /app/
+
+RUN pip install --upgrade pip && \
+    pip install --no-cache-dir -e .
+
+RUN apt-get update && apt-get install -y jq wget && \
```

Comment:
What if the system isn't apt-based?

Reply:
Our docker image is built from the base python:3 image which is Debian based

Reply:
⭕️